index.js

```diff
@@ -30,7 +30,7 @@ const gulpSass = (options, sync) => transfob((file, enc, cb) => { // eslint-disa
     return cb(null, file);
   }
 
-  const opts = clonedeep(options || {});
+  const opts = extend(true, options || {});
```

Comment:
*I think* we can drop the fallback `|| {}` since extend handles this cases.

Reply:
Makes sense